Source/Tools/FEXRootFSFetcher/Main.cpp

```diff
@@ -1114,6 +1127,38 @@ namespace UnSquash {
 
     return false;
   }
+
+  bool ExtractEroFS(const fextl::string &Path, const fextl::string &RootFS, const fextl::string &FolderName) {
+    auto TargetFolder = Path + FolderName;
+
+    bool Extract = true;
+    std::error_code ec;
+    if (std::filesystem::exists(TargetFolder, ec)) {
+      fextl::string Question = FolderName + " Already exists. Overwrite?";
+      if (AskForConfirmation(Question)) {
+        if (std::filesystem::remove_all(TargetFolder, ec) != ~0ULL) {
+          Extract = true;
+        }
+      }
+    }
+
+    if (Extract) {
+      ExecWithInfo("Extracting Erofs. This might take a few minutes.");
+
+      const auto ExtractOption = fmt::format("--extract={}", TargetFolder);
```

Comment:
Actually just tested, Since we're launching these applications with execvp, this works.
The only thing that didn't work was downloading the initial image directly with curl, which I fixed in another commit.